medusa/server/api/v2/scene_exception.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+"""Request handler for scene_exceptions."""
+
+from medusa.scene_exceptions import get_last_refresh, retrieve_exceptions
+from tornado.escape import json_decode
+from .base import BaseRequestHandler
+from .... import db
+
+
+def get_last_updates():
+    """Query the cache table for the last update for every scene exception source."""
+    last_updates = {}
+    for scene_exception_source in ['custom_exceptions', 'xem', 'anidb']:
+        last_updates[scene_exception_source] = get_last_refresh(scene_exception_source)[0]['last_refreshed']
+    return last_updates
+
+
+class SceneExceptionOperationHandler(BaseRequestHandler):
+    """Scene Exception request handler."""
+
+    def set_default_headers(self):
+        """Set default CORS headers."""
+        super(SceneExceptionOperationHandler, self).set_default_headers()
+        self.set_header('Access-Control-Allow-Methods', 'PUT')
+
+    def put(self):
```

Comment:
Actually you can use PUT or POST to create resources.  PUT **MUST** be idempotent, POST is **NOT** idempotent.